e2e-tests/e2e-tests.ts

```diff
@@ -6,14 +6,13 @@ let page
 let host = process.env.E2E_TEST_ENVIRONMENT
 
 before(async () => {
-  browser = await puppeteer.launch()
+  const browser = await puppeteer.launch({
+    executablePath: '/usr/bin/chromium-browser',
```

Comment:
shall we add some conditional here? this will not run locally (i.e. outside of docker)